### DISCUSSION TITLE
Create Wizard in modal

### DISCUSSION THREAD

**Discussion by warsemann** [filamentphp/filament#10166] [Help]
Hi its possible to run wizard in modal for create action?
Thanks

**Comment by leandrocfe** [comment]
Yes, use `->steps([])`
https://filamentphp.com/docs/3.x/actions/prebuilt-actions/create/#using-a-wizard

**Reply by devhoussam1998** [reply]
I use the wizard in modal but I couldn't edit the `->nexAction` and `->previousAction` does not exist for modal wizard

**Comment by warsemann** [comment]
Hi, when i use CreateAction its not in modal :(, i must use Action::make for that works.

I define CreateACtion in getHeaderActions() of  List Resource, it is the good way ?

**Reply by leandrocfe** [reply]
Yes, you can use this
